lib/private_api.rb

```diff
@@ -76,86 +117,56 @@ def format_order_hash(o)
 
 
 
-
-  # <----- BALANCE .
-
-  #Balance
-  def balance_for(asset='XBT', opt={})
-    puts "BitX.balance_for will be deprecated. please use BitX.balance to get a list of account balances"
-    opt.merge!({asset: asset})
-    path = '/api/1/balance'
-    r = authed_request(path, {params: opt})
-    j = JSON.parse(r.body)
-    raise BitXError.new("BitX balance error: #{j['error']}") if j['error']
-    balance = BigDecimal(j['balance'][0]['balance'])
-    reserved = BigDecimal(j['balance'][0]['reserved'])
-    {
-      asset:      asset,
-      balance:    balance,
-      reserved:   reserved,
-      available:  balance - reserved
+  # <----- TRANSACTIONS .
+  def transactions(account_id, min_row=1, max_row=100)
+    path = "https://api.mybitx.com/api/1/accounts/#{account_id}/transactions"
+    opt = {
+      min_row: min_row,
+      max_row: max_row
     }
+    authed_request(path, opt)
```

Comment:
The balances, available and delta fields should probably be converted to BigDecimal.